pkg/executor/sortexec/parallel_sort_spill_helper.go

```diff
@@ -52,14 +52,15 @@ func newParallelSortSpillHelper(sortExec *SortExec, fieldTypes []*types.FieldTyp
 		errOutputChan: errOutputChan,
 		finishCh:      finishCh,
 		fieldTypes:    fieldTypes,
-		tmpSpillChunk: chunk.NewChunkWithCapacity(fieldTypes, spillChunkSize),
+		tmpSpillChunk: chunk.NewChunkFromPoolWithCapacity(fieldTypes, spillChunkSize),
 	}
 }
 
 func (p *parallelSortSpillHelper) close() {
 	for _, inDisk := range p.sortedRowsInDisk {
 		inDisk.Close()
 	}
+	p.tmpSpillChunk.Destroy(spillChunkSize, p.fieldTypes)
```

Comment:
put it in defer block?

Reply:
I think it is impossible to have a problem with the ```isDisk.Close``` and lead to panic  here.

Reply:
OK